models/reservation.js

```diff
@@ -152,6 +98,21 @@ exports.finish = function(reservationId) {
     });
 };
 
+exports.review = function(reservationId, rating, comment) {
+  var reservation;
+  return exports.updateStatus(reservationId, exports.ReservationStatusEnum.reviewed)
+    .then(function(res) {
+      reservation = res;
+      return Promise.all([
+        Spot.review(reservation.attributes.spotId, reservationId, rating, comment),
+        User.removeReservationFromDriverActive(reservation)
+      ]);
+    })
+    .then(function() {
+      return reservation;
```

Comment:
I was just thinking the reservation's state would be changed, but I'm guessing the 'res' returned from updateStatus has the updated status? If nothing's changed then might as well just leave it